src/js/components/Diagram/doc.js

```diff
@@ -64,6 +68,16 @@ export const themeDoc = {
     type: 'string | {dark: string, light: string}',
     defaultValue: 'accent-1',
   },
+  'global.animation': {
```

Comment:
we should split it to `global.animation.duration`, `global.animation.draw`

Reply:
In reference to the next comment on the custom theme durations, I believe specifying here in the document that you can customize the attributes of the specific animations that are available to Diagram might relieve the confusion here. 

It took me a little bit of experimenting myself to realize that if you specify a `global.animation.duration` that will be a catch-all duration for any of the animations available for Diagram, and then `global.animation.draw` would be an object where you can specify specific attribute customizations to only that draw animation. I can aim to make this clearer in the `doc.js` for this component and I'll push it up to get some more thoughts.

Reply:
Since `global.animation.draw.duration` is specific to the diagram behavior only, we should have a theme prop on the diagram that provides `duration` of itself and it can fall back to the global animation value in case its `undefined`

Reply:
Updated the doc.js example, so now it's easy to understand that you can specify a duration under `global.animation`

Reply:
This prop was now moved to `diagram.animation` to keep it specific to diagram's behavior only.